src/App.js

```diff
@@ -1,23 +1,48 @@
+import {useState, useRef} from 'react';
 import styled from 'styled-components';
 
 import MagicForm from './components/MagicForm/MagicForm';
+import MagicList from './components/MagicList/MagicList.js';
+import {CodeSpan} from './components/Stylesheet/StyledSpans.js';
 
 export default function App() {
-  //
-  function analyzeSpell(spellword) {
+  // State // for the archived Spells
+  const [savedOrders, setSavedOrders] = useState([]);
+
+  //useRef is used here to find the MagicList at the End of following saveSpellOrder-function
+  const ref = useRef();
+
+  // ANALYZE SPELL // checks the Input.value from MagicForm and sends back and to: MagicList //
+  function saveSpellOrder(spellword) {
+    //
+    const trimmedInput = spellword.trim();
+    const checkedInput = trimmedInput.toLowerCase();
```

Comment:
Why don't you do this in one line, so spellWord.toLowerCase().trim()

Reply:
Was doing stuff step by step, but youre right, can do it in 1 step here. Really good, really sensible code review, thanks a lot